config/default/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+namespace: aws-provider-system
+namePrefix: aws-provider-
+
+resources:
+- ../crds/awsprovider_v1alpha1_awsclusterproviderconfig.yaml
+- ../crds/awsprovider_v1alpha1_awsclusterproviderstatus.yaml
+- ../crds/awsprovider_v1alpha1_awsmachineproviderconfig.yaml
+- ../crds/awsprovider_v1alpha1_awsmachineproviderstatus.yaml
+- ../rbac/rbac_role.yaml
+- ../rbac/rbac_role_binding.yaml
+- ../manager/manager.yaml
+
+patches:
+- ../../cmd/clusterctl/examples/aws/out/aws_manager_image_patch.yaml
+
+secretGenerator:
+- name: machine-controller-credential
```

Comment:
Suggest naming this `bootstrap-credentials` or `infra-provisioner-creds` as these aren't the creds that will be used by the machine controllers in the internal clusters.

Reply:
+1 changed